R/InsertTable.R

```diff
@@ -417,8 +417,8 @@ insertTable.DatabaseConnectorDbiConnection <- function(connection,
   }
   isSqlReservedWord(c(tableName, colnames(data)), warn = TRUE)
   
-  tableName <- gsub("^#", "", tableName)
-  if (dbms(connection) == "sqlite") {
+  #tableName <- gsub("^#", "", tableName) #EF: SQL Server needs the hashtag for temp tables
```

Comment:
I think this breaks behavior on all DBI drivers currently directly supported by DatabaseConnector? (SQLite, DuckDB)